### PR TITLE
[main][deploy] create script for adding pool to fairlaunch.

### PR DESCRIPTION
**Pull request by blackalpacaa**
## Description

- Create adding pool to FairLaunch script.
- Adjust set beneficial vault script.

## Why?

- To add new pool to FairLaunch.
- To set beneficial vault.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -159,9 +160,10 @@
     "deploy:testnet:oracles:config:add-price-source-chainlink": "hardhat --network testnet deploy --no-compile --reset --tags AddSourceChainLinkPriceOracle",
     "deploy:testnet:workers:upgrade:workers02": "hardhat --network testnet deploy --no-compile --reset --tags UpgradeWorkers02",
     "deploy:testnet:workers:config:set-treasury-config": "hardhat --network testnet deploy --no-compile --reset --tags TimelockAddTreasuryFieldsWorkers02",
-    "deploy:testnet:s028": "hardhat --network testnet deploy --no-compile --reset --tags TimelockAddBeneficialBuybackFieldsWorkers02",
+    "deploy:testnet:workers:config:set-beneficial-vault-config": "hardhat --network testnet deploy --no-compile --reset --tags TimelockAddBeneficialBuybackFieldsWorkers02",
```
Comment (alpanaca) [inline package.json:163]:
```suggestion
    "deploy:testnet:workers:config:set-beneficial-vault-config": "hardhat --network testnet deploy --no-compile --reset --tags TimelockSetBeneficialBuybackFieldsWorkers02",
```

File: deploy/exec/fairlaunch/config/add-pool.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
```
Comment (spicysquid168) [inline deploy/exec/fairlaunch/config/add-pool.ts:1]: You can remove this. I already implement it in another PR.

Comment (spicysquid168) [reply deploy/exec/fairlaunch/config/add-pool.ts:1]: I mean remove the whole deployment script.

**Comment by spicysquid168** [comment]
Close due to all scripts already done.

**Pull request closed without merging**